packages/sp/src/clientsidepages.ts

```diff
@@ -440,12 +475,46 @@ export class ClientSidePage extends File {
     /**
      * Get the liked by information for a modern site page     
      */
-    public getLikedByInformation(): Promise<void> {
+    public getLikedByInformation(): Promise<any> {
         return this.getItem().then(i => {
             return i.getLikedByInformation();
         });
     }
 
+    protected getCancasContent1(): string {
```

Comment:
Seems like a typo. Should it be `getCanvasContent1`?

Reply:
NO, in my country we spell it Cancas. Duh.

(yes, fixed it, thanks :) )